scripts/buildNomisIncidentTypePayload.ts

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env npx tsx
+
+/*
+Generates the JSON payload required by Prison API to update an incident type configuration in NOMIS.
+
+Usage:
+  ./scripts/buildNomisIncidentTypePayload.ts <dpsTypeCode>
+
+Where <dpsTypeCode> is one of the codes from server/reportConfiguration/constants/types.ts
+  e.g., DIRTY_PROTEST_1, ASSAULT_5, FIND_6, ...
+
+It prints to stdout the JSON body you can use with:
+  PUT ${HMPPS_PRISON_API_URL}/incidents/configuration/<NOMIS_CODE>
+*/
+
+import { printText } from './utils'
+import { types } from '../server/reportConfiguration/constants'
+import { getIncidentTypeConfiguration } from '../server/reportConfiguration/types'
+import type {
+  IncidentTypeConfiguration as DpsIncidentTypeConfiguration,
+  QuestionConfiguration as DpsQuestionConfiguration,
+} from '../server/data/incidentTypeConfiguration/types'
+
+interface Arguments {
+  scriptName: string
+  dpsTypeCode: string
+}
+
+main()
+
+async function main() {
```

Comment:
This builds the payload to send to NOMIS for an incident type to allow them to be created in this service and then sync'd to NOMIS